kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/BatchRestApi.java

```diff
@@ -32,6 +32,8 @@ public class BatchRestApi {
 
   private static final String API_BASE_PATH = "batches";
 
+  private BatchRestApi() {}
```

Comment:
why we need this?

Reply:
to disable user invoke the default constructor?

Reply:
The default constructor will cause npe since the client is not set, that means we must call the constructor which hold the KyuubiRestClient

Reply:
got it, thanks